Shared/IISFunctions/Get-IISModules.ps1

```diff
@@ -115,7 +159,15 @@ function Get-IISModules {
         }
     }
     process {
-        $modules = GetIISModulesSignatureStatus -Modules $ApplicationHostConfig.configuration.'system.webServer'.globalModules.add
+        $iisModulesParams = @{
+            Modules = $ApplicationHostConfig.configuration.'system.webServer'.globalModules.add
```

Comment:
When setting `Modules` variable, this is where you do something like `Where-Object { $SkipLegacyOSModulesCheck -and (in list of skip modules) }